hc/api/tests/test_create_check.py

```diff
@@ -44,26 +45,39 @@ def test_it_works(self):
         self.assertEqual(check.tags, "bar,baz")
         self.assertEqual(check.timeout.total_seconds(), 3600)
         self.assertEqual(check.grace.total_seconds(), 60)
+        self.assertEqual(check.last_ping,None)
+        self.assertEqual(check.n_pings,0)
 
     def test_it_accepts_api_key_in_header(self):
-        payload = json.dumps({"name": "Foo"})
+        payload = json.dumps({"name": "Foo", "api_key": "abc"})
 
         ### Make the post request and get the response
         r = {'status_code': 201} ### This is just a placeholder variable
 
-        self.assertEqual(r['status_code'], 201)
+        r = self.client.post(self.URL, payload,
+                             content_type="application/json")
+        self.assertEqual(r.status_code, 201)
 
     def test_it_handles_missing_request_body(self):
+        payload = json.dumps({"name": "Foo", "api_key": "efg"})
+        
         ### Make the post request with a missing body and get the response
         r = {'status_code': 400, 'error': "wrong api_key"} ### This is just a placeholder variable
```

Comment:
This line is longer than the recommended pep8 standard of 75 characters. The comment here may not be necessary